docs/community/com-config/com-config-flags.md

```diff
@@ -9,64 +9,63 @@ grand_parent: Community
+| [translation.map-size](#translation-map-size)           | FEATUREBASE_TRANSLATION_MAP_SIZE                  | int   | Size in bytes of mmap to allocate for key translation.     |
+| [auth.enable](#auth-authenticationauthorization-configuration)                    | FEATUREBASE_AUTH_ENABLE                           | bool  | Enable AuthN/AuthZ of featurebase, disabled by default.     |
+| [auth.client-id](#auth-authenticationauthorization-configuration)                 | FEATUREBASE_AUTH_CLIENT_ID                        | str   | Identity Provider's Application/Client ID.     |
+| [auth.client-secret](#auth-authenticationauthorization-configuration)             | FEATUREBASE_AUTH_CLIENT_SECRET                    | str   | Identity Provider's Client Secret.     |
+| [auth.authorize-url](#auth-authenticationauthorization-configuration)             | FEATUREBASE_AUTH_AUTHORIZE_URL                    | str   | Identity Provider's Authorize URL.     |
+| [auth.token-url](#auth-authenticationauthorization-configuration)                 | FEATUREBASE_AUTH_TOKEN_URL                        | str   | Identity Provider's Token URL.     |
+| [auth.group-endpoint-url](#auth-authenticationauthorization-configuration)        | FEATUREBASE_AUTH_GROUP_ENDPOINT_URL               | str   | Identity Provider's Group endpoint URL.     |
+| [auth.redirect-base-url](#auth-authenticationauthorization-configuration)         | FEATUREBASE_AUTH_REDIRECT_BASE_URL                | str   | Base URL of the featurebase instance used to redirect IDP.     |
+| [auth.logout-url](#auth-authenticationauthorization-configuration)                | FEATUREBASE_AUTH_LOGOUT_URL                       | str   | Identity Provider's Logout URL.     |
+| [auth.scopes](#auth-authenticationauthorization-configuration)                    | FEATUREBASE_AUTH_SCOPES                           | list  | Comma separated list of scopes obtained from IdP     |
+| [auth.secret-key](#auth-authenticationauthorization-configuration)                | FEATUREBASE_AUTH_SECRET_KEY                       | str   | Secret key used for auth.     |
+| [auth.permissions](#auth-authenticationauthorization-configuration)               | FEATUREBASE_AUTH_PERMISSIONS                      | str   | Permissions' file with group authorization.     |
+| [auth.query-log-path](#auth-authenticationauthorization-configuration)            | FEATUREBASE_AUTH_QUERY_LOG_PATH                   | str   | Path to log user queries     |
+| [auth.configured-ips](#auth-authenticationauthorization-configuration)            | FEATUREBASE_AUTH_CONFIGURED_IPS                   | str   | List of configured IPs allowed for ingest     |
+| [future.rename](#future-rename)            |   | str   | Enables use of Featurebase_ environment variables for flags.     |
```

Comment:
This feels weird here. Also it's not linking anywhere on the page that I can see

Reply:
Removed from here and added it in the warning!